docs/content/mobile/android/local-deploy.md

```diff
@@ -17,6 +17,9 @@ To connect your Android device to a local deployment, you need to prepare its co
 1. Select **Save** and close the window.
 
 ## Unity Editor{#in-editor}
+1. If your project does not already contain one, create a MonoBehaviour script which inherits from the [`MobileWorkerConnector`](https://github.com/spatialos/gdk-for-unity/blob/master/workers/unity/Packages/com.improbable.gdk.mobile/Worker/MobileWorkerConnector.cs) and include the functionality you want. <br> 
+In your Unity Editor, add this script it to your Android client-worker GameObject. In the FPS Starter Project this is located at **Assets** > **Fps** > **Prefabs** > **AndroidClientWorker**.
```

Comment:
add this as a separate step. 
1. create monobehaviour
2. add it to your game object

Reply:
This looks good. Two separate steps reads slightly better imo.